test/test_helper.rb

```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
+require 'bundler/setup'
+
 $LOAD_PATH.unshift(File.dirname(__FILE__) + '/../lib')
 
 require 'puma-acme'
 
-require 'http'
+require 'HTTP'
```

Comment:
(°_o) this looks strange! I trust you, @geemus, but wow. Huh.

Reply:
it fails otherwise (that's what was happening on main). FWIW, requiring 'http' is what you do to get the http gem vs requiring 'HTTP' to get the http.rb gem. I think they both use the same namespace too...